### DISCUSSION TITLE
How to set OCG layer’s Locked state to True/False

### DISCUSSION THREAD

**Discussion by ranga-tolapi** [py-pdf/pypdf#1773] [General]
How to set OCG layer’s Locked state to True/False. So that, it shall reflect in the layer properties in adobe viewer. Please refer to the following screenshot.

![5064F242-5B0A-417F-8FF3-53D2F55622B7](https://user-images.githubusercontent.com/17163418/230751680-e2832b6a-bafa-4daa-a494-395f9079d7fc.jpeg)

**Comment by MartinThoma** [comment]
pypdf does not support OCG layers, see https://github.com/py-pdf/pypdf/discussions/1181
